docs/Links2GoogleDocs.md

```diff
@@ -0,0 +1,40 @@
+# Links to google docs
+
+### General 
+- [Write Flows & Data Policy Granularity Options](https://docs.google.com/document/d/1rQ4ykFfOZ4mfD6rrd1XgInKSiJ0DPBoHu9xwawJecFI/edit)
+- [Injest Flow](https://docs.google.com/document/d/1j5NBs-xN8y8hxM01g2ZwgGvROQGIV-yqk-LwbJsfUoo/edit)
+
+### Multi Cluster
+- [Multicluster architecture](https://docs.google.com/document/d/1r5VLIqV07YharGTlaN7-Dbo33Mdy7AgEg29ArqmKSTo/edit)
+
+### Smart Blueprints
+- [Component Overview](https://docs.google.com/document/d/19JBTm3UOJ7K2MJcItRw3ViKcAd2jiobh8_-xUlQT42A/edit#heading=h.eglyswnpsl1n)
+- Structure 
+  - [Smart blueprint structure design](https://docs.google.com/document/d/1Qu5OwiozPy5dr2l75FBPbzZsRygUkLSKD0tJ74Wzfws/edit#heading=h.iahrcyqumdcr)
+  - [Test Scenarios](https://docs.google.com/document/d/1vsiFiVvc2Al8dIN5TopHZPd43-fkpCFT1qgS97hqSvA/edit)
+- Optimizer 
+  - [Optimization Requirements](https://docs.google.com/document/d/1eYxi3tiH4peOnzs74KyAXuCvP8NKhGLkQkRJRSGgRxg/edit#heading=h.lt47hx98pfln)
+  - [Design](https://docs.google.com/document/d/171TWD3FPEV-ZNoO_f8vQQoj-EV-wy6kuq6IyQn6QPPY/edit#heading=h.o9149roj6q02)
+- Config 
+  - [Requirements](https://docs.google.com/document/d/1xG5kQOYMs6OzovUVOzh9t_c68QlaGtRDCpo34dCm-nA/edit)
+  - [Design](https://docs.google.com/document/d/1l7TmlacPjXGQJ_83OiJHYa40P_SL-YiQgxAcZqzyr_M/edit)
+  - [QuickStart Scenario](https://docs.google.com/document/d/1Tt8OEDifGJmmtzGmr9yloTHhtEjdPw2g6Io6aHeTeYg/edit#heading=h.awukm0nz77st)
+
+### Distributed Credentials
+- [Credentials](https://docs.google.com/document/d/1MA9Oa1aqUidTSBEh79lF8KL6pOhxChbVnD5wjyhAYec/edit#heading=h.lt47hx98pfln)(Outdated)
+- [Implementation](https://docs.google.com/document/d/1zcEMm-geg6QrSrIYuJxgFCDVdXiGNQVB65Llv3i_02s/edit#heading=h.img5h5xmx8ok)
+
+### Logging
+- [Fybrik Visibility Requirements and Design](https://docs.google.com/document/d/1x3Jnp3CIN6YhJKtWM1uIimaw9-q6Hb9M7LqkazmBRBo/edit)
+
+### Taxonomy
+- [Mesh for Data: taxonomy](https://docs.google.com/document/d/1sfdcNI6PTIYWsSIznpNqJNQCdVaVFchYB2vljzUrlew/edit#heading=h.img5h5xmx8ok)
+- [Taxonomy Technical Design](https://docs.google.com/document/d/1J7f8co97C5AXtpbgUKlSPCDMU60lkYmQnDEbx-aKuOM/edit)
+- [OPA Request & Result Structures](https://docs.google.com/document/d/1YjMWZyZd9rQvRU8TcQzg6E1zCYfA_cZ5wuUCJier4oA/edit#heading=h.px96c317wfnw)
+
+### Read Modules Endpoints
+- [Configuring Application](https://docs.google.com/document/d/10qL_sUBqbSmPww1GQ76ON8KjQjFVByU3aXarmCx9uHA/edit)
+- [Configuring connectivity and network isolation](https://docs.google.com/document/d/1KzkzzE4BnxnZb6PEiJOK3ol3bygt8N24z7V8CfV0Pd8/edit)
+
+### Storage management
+- [Dynamic Bucket Allocation](https://docs.google.com/document/d/1PzaQtED9NUq10REfxl0wMFOl247SKc0XvAUHpxjSYjY/edit#heading=h.img5h5xmx8ok)
```

Comment:
Please add the KFP component: https://drive.google.com/file/d/1xn7pGe5pEAEZxnIzDdom9r7K6s78alcP/view?usp=sharing

Reply:
done

Reply:
Hmm, I don't see it